src/main/core/support/configuration.rs

```diff
@@ -313,7 +313,7 @@ impl Default for ExperimentalOptions {
             use_cpu_pinning: Some(true),
             interpose_method: Some(InterposeMethod::Ptrace),
             runahead: None,
-            scheduler_policy: Some(SchedulerPolicy::Steal),
+            scheduler_policy: Some(SchedulerPolicy::Host),
```

Comment:
Is the `[default: "host"]` cli help string suffix auto-updated with these changes?

Reply:
Yup!